embed/etcd.go

```diff
@@ -66,13 +66,13 @@ const (
 type Etcd struct {
 	Peers            []*peerListener
 	Clients          []net.Listener
+	clientServers    map[string]*serveCtx
```

Comment:
why this is renamed?

Reply:
Wanted to make it clear that it handles client requests.

Reply:
this is a map that contains the context of the server that handles the client requires.

the important part is that it is a map of context. so we cannot really get rid of the ctx part.

i would rather keep the previous name, and add comments to the field.

Reply:
Got it. Just reverted that name change.
Will merge after test passes. Thanks.